### ISSUE TITLE
Geocoding addresses for building locations on centroids instead of streets.

### ISSUE THREAD

**Issue by zhik** [chriswhong/nyc-311-digest#14]
It would be great to put 311 complaints with `Address Type` of ADDRESS and building specific complaints on the building's centroids instead on the streets. This might require building a new API / data pipeline, or querying a geocoder API. 

![image](https://user-images.githubusercontent.com/9166469/170381987-51580156-b372-4e31-bd75-808cf424e375.png)

**Comment by chriswhong** [comment]
I agree, and this may be possible with a static bbl + centroids lookup
table.  I’ll see what I can do.
